harper-core/src/linting/pronoun_inflection_be.rs

```diff
@@ -51,6 +51,14 @@ impl PronounInflectionBe {
         map.insert(arent, "isn't");
 
         let is = SequenceExpr::default()
+            .then(|tok: &Token, _: &[char]| {
+                tok.kind
+                    .as_word()
+                    .as_ref()
+                    .and_then(|m| m.as_ref().and_then(|m| m.np_member))
```

Comment:
Ooh both the `.and_then` and the `np_member` are quite novel to me. I'd seen the latter but wasn't sure how to make use of it. The former I don't recall seeing before.

Since you were recently working on the documentation, did you cover these? Maybe they're "advanced"?